avocado/core/runner.py

```diff
@@ -359,6 +359,16 @@ def sigtstp_handler(signum, frame):     # pylint: disable=W0613
 
         signal.signal(signal.SIGTSTP, sigtstp_handler)
 
+        # Saving the original sys.path state
+        _syspath = sys.path[:]
+
+        # Injecting the test module into the sys.path, so user can define
+        # a method from the test module to be used in func_at_exit
+        if 'modulePath' in test_factory[1]:
+            test_path = test_factory[1]['modulePath']
+            test_module_dir = os.path.abspath(os.path.dirname(test_path))
+            sys.path.insert(0, test_module_dir)
```

Comment:
Well the important thing for me is to avoid modifying `sys.path` while running the test. Other solutions are welcome and as I said, the target usage is `self.runner_queue.func_at_exit(...)` which would be a method of `runner_queue` object, which would simply pickle the stuff and send it via itself to the runner. The thing is that `runner_queue` is still only a `SimpleQueue` because we haven't created this API yet. But more and more I think it'd be that simple that we should go ahead and do it...